controller/rebuild.go

```diff
@@ -166,17 +181,5 @@ func (c *Controller) PrepareRebuildReplica(address string) ([]string, error) {
 		return nil, err
 	}
 
-	fromHead := rwChain[0]
-
-	chain, err := getReplicaChain(address)
-	if err != nil {
-		return nil, err
-	}
-	toHead := chain[0]
-
-	if err := syncFile(fromHead+".meta", toHead+".meta", rwReplica, replica); err != nil {
-		return nil, err
-	}
-
-	return rwChain[1:], nil
+	return rwChain, nil
```

Comment:
We can skip prepare rebuild from WO replica, as we already have healthy replica info, so we can get the chain info from there, calling controller to perform rebuilding seems to be unnecessary as controller is not maintaining info.